tests/unit/OmiseTest.php

```diff
@@ -118,4 +128,54 @@ public function testGetContent_merchantSaveSetting_theSettingDataMustBeSaved()
 
         $this->omise->getContent();
     }
+
+    public function testHookPayment_moduleIsActivatedAndTheSettingOfModuleStatusIsEnabled_displayThePaymentTemplateFile()
+    {
+        $this->omise->active = true;
+        $this->setting->method('isModuleEnabled')->willReturn(true);
+        $this->omise->method('display')->willReturn('payment_template_file');
+
+        $this->assertEquals('payment_template_file', $this->omise->hookPayment(''));
+    }
+
+    public function testHookPayment_moduleIsActivatedAndTheSettingOfModuleStatusIsEnabled_displayCheckoutForm()
+    {
+        $this->omise->active = true;
+        $this->setting->method('isModuleEnabled')->willReturn(true);
+        $this->setting->method('getTitle')->willReturn('title_at_header_of_checkout_form');
+        $this->checkout_form->method('getListOfExpirationYear')->willReturn('list_of_expiration_year');
+
+        $this->smarty->expects($this->exactly(2))
+            ->method('assign')
+            ->withConsecutive(
+                 array('list_of_expiration_year', 'list_of_expiration_year'),
```

Comment:
@nimid just a little little, but can you remove 1 space from these 3 lines (151-153) while you update code. Thanks. 😅